infra/main.bicep

```diff
@@ -572,6 +595,7 @@ output AZURE_SEARCH_SERVICE string = searchService.outputs.name
 output AZURE_SEARCH_SECRET_NAME string = useSearchServiceKey ? searchServiceSecretName : ''
 output AZURE_SEARCH_SERVICE_RESOURCE_GROUP string = searchServiceResourceGroup.name
 output AZURE_SEARCH_SEMANTIC_RANKER string = actualSearchServiceSemanticRankerLevel
+output AZURE_SEARCH_SERVICE_ASSIGNED_USERID string = searchService.outputs.principalId
```

Comment:
I don't believe this is actually user assigned identity - seems like system assigned

Reply:
Yes, in core/infra/searchServices it's hardcoded to systemAssigned. Also this is not supported in free tier so this will break

Reply:
Free tier we cannot use integratedvectorization. I will leave it as it is. If free tier its better users dont set int_vectorizer flag. Updated readme with the same.